app/services/settings-v2/video.ts

```diff
@@ -240,6 +240,15 @@ export class VideoSettingsService extends StatefulService<IVideoSetting> {
 
   @debounce(200)
   updateObsSettings(display: TDisplayType = 'horizontal') {
+    // confirm all vertical fps settings are synced to the horizontal fps settings
+    if (display === 'vertical') {
+      this.syncFPSSettings();
```

Comment:
not sure i understand this-- so every time the vertical display is updated we make the vertical fps equal to the horizontal fps?

Reply:
Yes, this is the actual cause of the bug. If the fps settings are not the same for both contexts, the output settings is working with mismatched values, which creates this issue with speed and duration being out of sync. When a context is created, the dual output service syncs the vertical fps settings with the horizontal one. But any time we make a change to the fps settings, we need to apply this change to both contexts to keep them synced.